packages/itwinui-react/src/core/Menu/MenuItem.tsx

```diff
@@ -134,6 +142,11 @@ export const MenuItem = React.forwardRef((props, forwardedRef) => {
       return;
     }
 
+    // If DropdownMenuCloseOnClickContext's value = true, should close the `DropdownMenu` when the `MenuItem` is clicked
+    if (dropdownMenuCloseOnClickContext) {
```

Comment:
nit: This comment can be avoided if we make the variable name self-explanatory.

`dropdownMenuCloseOnClickContext` ➡️ `shouldCloseMenuOnClick`